### PR TITLE
second pr

### PR DESCRIPTION
**Pull request by Yexiansen66**
update fusen.md

### CODE REVIEW

**Review by Akshay2996 — CHANGES_REQUESTED** [review]
Hi @Yexiansen66 Please remove line 7, line 10 & line 13 from your fusen.md because this is not required it's just stated where the changes are made.

For Team.md please remove the merge conflict. You can follow along this [issue](https://github.com/twindle-co/twindle/issues/296)

**Comment by Yexiansen66** [comment]
@Akshay2996  I changed it.Is it OK?

**Review by naveenkala — CHANGES_REQUESTED** [review]
Remove line 33, 35 and 37 from TEAM.md.
Add | at the end of your introduction line.
EX -  .....more advanced knowledge.|

**Comment by Yexiansen66** [comment]
@naveenkala I changed it.is it right?

**Comment by Yexiansen66** [comment]
@naveenkala I changed it,Is it right?

**Review by naveenkala — APPROVED** [review]

**Review by tusharkandpal — APPROVED** [review]
👍🏻👍🏻

File: docs/TEAM.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
 |[Maria Rivera](https://github.com/GorettiRivera) | I have basic knowledge of Javascript, HTML and CSS. Currently working in different open source projects|
 |[Neaz Mahmood](https://github.com/Neaz-Mahmood)|I am from Dhaka, Bangladesh. I am a second year Computer Science undergraduate. I know basic html, css, javascript. I am here to learn & contribute to 'twindle'. This is my first open source project.|
 |[Predrag](https://github.com/stamenkovic-dev)| I'm from Serbia, and I have only basic knowledge of HTML and CSS, but I'm here to learn more about web development and team play at coding.|
+|[Fusen Ye](https://github.com/Yexiansen66)| I'm interested in programming.I known some HTML,CSS and JavaScript.I want to do something for twindle.And I want to learn more advanced knowledge.
```
[the remark this answers is not in the document]
Comment (Yexiansen66) [reply docs/TEAM.md]: I'm confused, I don't know how to do it.Can you give me some guidance？

File: firstpr/fusen.md [hunk 2]
```diff
@@ -2,4 +2,12 @@ I am Fusen Ye from China.
...
\ No newline at end of file
+if I make some mistakes,please forgive me.
+
+<<<<<<< HEAD
+//second pr
+I like hot weather.
+=======
+second pr
+Make a little progress every day.
+>>>>>>> 3f7c6f3359432cd377174e4c22809592cc80b62d
```
[the remark this answers is not in the document]
Comment (Yexiansen66) [reply firstpr/fusen.md]: I removed these in my fork.Does the problem still exist？I think I need some help.

[the remark this answers is not in the document]
Comment (tusharkandpal) [reply firstpr/fusen.md]: Its ok now

[the remark this answers is not in the document]
Comment (Yexiansen66) [reply firstpr/fusen.md]: Can it merge?

**Review by shekhar10feb — APPROVED** [review]

File: docs/TEAM.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tusharkandpal) [reply docs/TEAM.md]: sorry for replying late... btw you did it already 😄 
you're good to go 👍

File: firstpr/fusen.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tusharkandpal) [reply firstpr/fusen.md]: yes 👍

**Review by tusharkandpal — APPROVED** [review]

**Pull request merged**